src/routes/v1/friendships/types.rs

```diff
@@ -24,20 +24,20 @@ pub struct FriendshipFriend {
 
 #[derive(Debug, Default, Serialize, Deserialize)]
 pub struct MessageRequestEventResponse {
-    pub friendships: Vec<MessageRequestEvent>,
+    pub messages_req_events: Vec<MessageRequestEvent>,
```

Comment:
just a comment to let you know that you can do something like the example below when we want another name for the internet.
```rust
pub struct MessageRequestEventResponse {
 #[serde(rename = "messages_request_event")]
 pub messages_req_events: Vec<MessageRequestEvent>,
}
```

check out https://serde.rs/field-attrs.html

Reply:
Amazing! thanks 🫶🏼